services/app-web/src/components/HealthPlanPackageTable/HealthPlanPackageTable.tsx

```diff
@@ -249,11 +310,14 @@ export const HealthPlanPackageTable = ({
                     {showFilters && (
                         <FilterAccordion
                             onClearFilters={() => {
+                                // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                                //@ts-ignore
```

Comment:
what does typescript complain about here?

Reply:
`type "never" has no call signature`  jotai types the set function as a `never`  I'm not sure why.

Reply:
To be clear, there are a bunch of issues/discussions in jotai about this, and I've tried the various proposed solutions; no luck so far, but I think it's fixable with the correct incantation.